From e273bd588929d15badd04a3d9168b6f804d4cd03 Mon Sep 17 00:00:00 2001
From: "Ranjan, Rajani" <rajani.ranjan@intel.com>
Date: Wed, 26 Apr 2023 14:18:20 +0530
Subject: [PATCH] Sepolicy changes to support sensors aidl hal.

configuring sensors service by providing correct label
to start ctl.interface_start for aidl
android.hardware.sensors@aidl-service.intel

Tracked-On: OAM-110600
Signed-off-by: Ranjan, Rajani <rajani.ranjan@intel.com>
---
 sensors/mediation/file_contexts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sensors/mediation/file_contexts b/sensors/mediation/file_contexts
index 16db756..8d6d7e5 100644
--- a/sensors/mediation/file_contexts
+++ b/sensors/mediation/file_contexts
@@ -1,2 +1,2 @@
 # sensors HAL service
-/vendor/bin/hw/android\.hardware\.sensors@[0-9]+.?[0-9]*-service.intel  u:object_r:hal_sensors_default_exec:s0
+/vendor/bin/hw/android\.hardware\.sensors@aidl-service\.intel  u:object_r:hal_sensors_default_exec:s0
-- 
2.41.0

